src/index.html

```diff
@@ -4,10 +4,58 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__top">
+        <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="Product photo"
+        >
+      </div>
+
+      <div class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__product-code">
+        Product code: 195434
+      </div>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__reviews">
+          Reviews: 5
+        </div>
+      </div>
+
+      <div class="card__price-block">
```

Comment:
```suggestion
      <p class="card__price-block">
```

Reply:
I have a block in which there were 2 elements, I can't break <p>

Reply:
> I have a block in which there were 2 elements, I can't break <p>